portfolio/src/main/webapp/index.html

```diff
@@ -11,6 +11,28 @@
       <h1>Amy's Portfolio</h1>
       <p>Hi! I'm Amy and welcome to my portfolio!</p>
       <p>I'm a rising junior at UC San Diego studying Data Science!</p>
+      <h2>Background</h2>
+      <p>I am a four-time instructional assistant for the data structures 
+         and algorithms course at my university. I absolutely love it 
+         because I get to help people learn about my favorite subject 
+         and I find it extremely rewarding being able to help information 
+         click for someone.
```

Comment:
I am not sure what "help information click for someone" means. Can you rephrase?

Reply:
Yes, I reworded it to say "help students finally understand the material."